docs/sources/userguide/cluster.md

```diff
@@ -0,0 +1,307 @@
+page_title: Getting started with Docker Cluster
+page_description: Introductory guide to getting a Docker cluster setup
+page_keywords: documentation, docs, cluster, docker multi host, scheduling
+
+# Getting Started with Docker Cluster
+
+
+This section provides a quick introduction to getting a Docker cluster setup
+on your infrastructure.
+
+## Discovery
+
+Before we can start deploying our container's to a Docker cluster we need to 
+ensure that each of the nodes are able to discover each other.  The Hub
+allows you to manage your Docker cluster easily.  To start a new cluster 
+login to your Hub account and hit **Create Cluster** providing a name
+of your choosing.  Under my account I will create a new cluster named
+**cluster-1**.  After creating a cluster you should be able to see and manage 
+the nodes that are currently registerd.  After creating a cluster you 
+should receive a URL that looks similar to 
+`https://discovery.hub.docker.com/u/demo-user/cluster-1` with your Hub 
+username and cluster name.
+
+
+To add a new or existing Docker
+Engine to your newly created cluster use the provided URL from the Hub
+with the `--discovery` flag when you start Docker in daemon mode.
+
+    $ docker -d --discovery https://discovery.hub.docker.com/u/demo-user/cluster-1
+
+
+In the future, other discovery mechanism will be provided that will not depend on the Hub.
+
+## Master
+
+In order to ensure consistency within the cluster one of your Docker Engines
+will need to be promoted to master within the cluster.  If you are using
+the Hub's discovery service you will be able to promote any of your registered
+nodes to a master from the web interface.  You can also statically assign one
+of the Docker Engines with the `--master` flag.
+
+    $ docker -d --master --discovery https://discovery.hub.docker.com/u/demo-user/cluster-1
+
+Eventually, clustering will provide a built-in leader election algorithm making the `--master` flag obsolete.
```

Comment:
What are the available options for allowing fault tolerance in cluster ?(If master goes down.. one of the nodes has to be picked and given privileges). I want to use Docker swarm orchestration technology and docker swarm doesn't provide any fault tolerance.